src/app/static/about/about.component.html

```diff
@@ -86,152 +86,152 @@ <h2>{{ 'anms.about.contributors' | translate }}</h2>
     <div class="contributors-wrapper">
 
       <!-- ALL-CONTRIBUTORS-LIST:START -->
-  <a class="contributor" href="https://medium.com/@tomastrajan" target="_blank">
-    <img src="https://avatars0.githubusercontent.com/u/3764868?v=4">
+  <a class="contributor" href="https://medium.com/@tomastrajan" target="_blank" attr.aria-label="{{'anms.a11y.about.contributors-link' | translate}}">
```

Comment:
I think one translation for contributors is fine. But this shouldn't be handled here because this is HTML is generated with the `contributors:generate` command. To solve this we'll have to edit `.all-contributors-html` to include the aria label.

If we want to get fancy we can also generate an array of contributors in `about.component.ts` and loop over this array in the html to create an aria label saying: `Open link to contributor {{name}} contact page`.

To do this, we have to do:

Create the array in the component file `about.component.ts`:
```ts
contributors = [
    // ALL-CONTRIBUTORS-LIST:START
    // ALL-CONTRIBUTORS-LIST:END
];
```

Loop over this array using `*ngFor` in `about.component.html`:
```html
<a *ngFor="let c of contributors" class="contributor" [href]="c.profile" target="_blank" [attr.aria-label]="'anms.a11y.about.contributors-link' | translate:{name: c.name}">
        <img [src]="c.avatar_url">
        <span>{{c.name}}</span>
</a>
```

Edit the generate script to:
```ts
const data =
  contributors.map(
    c => `
{
  profile: '${c.profile}',
  avatar_url: '${c.avatar_url}',
  name: '${c.name}',
}`
  ) + '\n';
```

Thoughts @geromegrignon and @tomastrajan ? 😁

Reply:
I'll work on it tomorrow

Reply:
@timdeschryver @geromegrignon please can you elaborate what would be the difference with array approach? `.all-contributors-html` can generate any html (including aria stuff) which then gets processed during build / runtime ?